src/public.js

```diff
@@ -249,6 +249,36 @@ Grid.defaults = {
             // default converter
             from: function (value) { return value; },
             to: function (value) { return value; }
+        },
+        timestamp: {
+            to: function(value) {
+                return (new Date((+value) * 1000)).toLocaleString(navigator.language, { timeZone: 'UTC' });
```

Comment:
maybe use momentjs

Reply:
For one line? That seems like overkill to me. (Incidentally, I have taken this code from a PR you created against the `core` repo.)

Reply:
Ok, maybe I've seen it too late too ;)

It is in core anyway: https://github.com/opnsense/core/blob/master/src/opnsense/www/js/moment-with-locales.min.js